articles/user-guide/host-programs.md

```diff
@@ -0,0 +1,563 @@
+---
+title: Run Q# standalone or with host programs
+description: Overview of the different ways to run Q# programs: command line, Q# Jupyter Notebooks, and classical host programs in Python or a .NET language.
+author: gillenhaalb
+ms.author: a-gibec@microsoft.com
+ms.date: 05/15/2020
+ms.topic: article
+uid: microsoft.quantum.guide.host-programs
+---
+
+# Run Q# standalone or with host programs
+
+One of the Quantum Development Kit's greatest strengths is its flexibility across platforms and development environments.
+However, this also means that new Q# users may find themselves confused or overwhelmed by the numerous options found in the [install guide](xref:microsoft.quantum.install).
+On this page, we explain what happens when a Q# program is run, and compare the different ways in which users can do so.
+
+A primary distinction is between running Q#
+    - as a standalone application, where Q# is the only language involved and the program is invoked directly, or
```

Comment:
```suggestion
    - with the command line interface as a standalone application, where Q# is the only language involved and the program is invoked directly, or
```

Reply:
Same as comment below; switching to this would leave out Jupyter

Reply:
If we want to adopt the suggestion, we could break IQ# Notebooks out into their own third line. That said, I agree, we need to keep IQ# Notebooks front-and-center in this list.

Reply:
Tried a middle ground in new commit: command line and NBs still under umbrella of "standalone applications", but brought both out explicitly with a sublist